cms/models.py

```diff
@@ -76,11 +117,24 @@ def save(self, *args, **kwargs):
         self.slug = slugify("bootcamp-{}".format(self.bootcamp_run.run_key))
         super().save(*args, **kwargs)
 
+class LearningAreasPage(BootcampProductChildPage):
+    """
+    Learning area's page for learning benefits.
+    """
+    learning_areas = StreamField(
+        [("learning_areas", LearningAreaBlock())],
+        blank=False,
+        help_text="Enter detail about learning areas upto max 3 blocks.",
```

Comment:
This needs to be _exactly_ 3 blocks, not _up to_ 3 blocks.

Reply:
Please add `min_num=3, max_num=3`. We need exactly 3 blocks, not up to 3 blocks here.